src/main/java/nus/climods/ui/MainWindow.java

```diff
@@ -184,6 +186,9 @@ private void handleExit() {
         GuiSettings guiSettings = new GuiSettings(primaryStage.getWidth(), primaryStage.getHeight(),
             (int) primaryStage.getX(), (int) primaryStage.getY());
         logic.setGuiSettings(guiSettings);
+        PauseTransition delay = new PauseTransition(Duration.seconds(2));
```

Comment:
2 second of delay feels quite long and makes the app looks like its lagging

Reply:
aight will change it to 0.7 seconds instead